packages/block/src/helpers.ts

```diff
@@ -152,16 +146,15 @@ export const getDataGasPrice = (header: BlockHeader) => {
 }
```

Comment:
I am still a bit shaky here on all the changes from this whole price calculation, but if I grasp things correctly I think code docs and error msgs from the `getDataGasPrice()` helper method above with all this parent references might not be correct.

Reply:
ahh yes, will update, as the dataprice is no more based on parent, but the current header